contrib/tinks/res/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
So git creates the folders for the generated stuff, it replaces the `mkdir -b bin res` from the Makefile. :wink: